vault/resource_identity_group.go

```diff
@@ -101,7 +100,14 @@ func identityGroupResource() *schema.Resource {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Default:     false,
-				Description: "Manage member entities externally through `vault_identity_group_policies_member_entity_ids`",
+				Description: "Manage member entities externally through `vault_identity_group_member_entity_ids`",
+			},
+
+			"external_member_group_ids": {
```

Comment:
Do we need to update the resource docs for this field?

Reply:
Yup! Thanks for catching that 😅 Added docs in latest commit